environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -20,11 +20,12 @@ import (
 type Container struct {
 	ID           string
 	statsChannel chan *docker.Stats
+	doneChannel chan bool
```

Comment:
But why do you need to close this channel multiple times? You just need to close it once, when we want to stop the collection.